### PR TITLE
IDVA6-1664 Wrong article used in confirmation for adding a user for ACSP

### PR DESCRIPTION
**Pull request by mmodzelewski-ch**
Link to Jira:
https://companieshouse.atlassian.net/browse/IDVA6-1664

Changes:
- Strings in both sides of the condition changed to lower case to ensure correct result of comparison 
- Tests refactored to increase coverage

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/confirmation-member-added.njk [hunk 1]
```diff
@@ -3,35 +3,39 @@
 
 {% extends "layouts/default.njk" %}
 
-{% set userDetails = newUserDetails.userName if newUserDetails.userName else newUserDetails.email %}
-{% set roleText = lang.as_a + userRole if userRole == lang.standard_user else lang.as_an + userRole%}
+{% set userDetails = newUserDetails.userName if newUserDetails.userName else 
+    newUserDetails.email %}
+{% set roleText = lang.as_a + userRole if (userRole | lower === lang.standard_user | lower)
```
Comment (tferns-ch) [inline src/views/confirmation-member-added.njk:8]: Looks good but it was a little difficult to see these were the only "relevant" changes as a lot of the other indentation on the file was changed.

**Review by tferns-ch — APPROVED** [review]
Looks good, nice work on the tests.  It looks like some of the indentation is off on the html file, it might be good to ensure all elements are aligned properly and have the right level of indentation - perhaps we can add something to husky to format/lint this for us? 

Anyways, nice work.

**Review by bwallace-ch — APPROVED** [review]

File: src/views/confirmation-member-added.njk (hunk 1 above)
Comment (mmodzelewski-ch) [reply src/views/confirmation-member-added.njk:8]: Ah, yes, sorry for that. My VS Code is configured to automatically reformat code on save and it applies the formatting to the whole file.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**